docs/noencap-hybrid-modes.md

```diff
@@ -14,7 +14,8 @@ NetworkPolicy enforcement, which is why trying to disable AntreaProxy in these
 modes will normally cause the Antrea Agent to fail. It is possible to override
 this behavior and force AntreaProxy to be disabled by setting the
 ALLOW_NO_ENCAP_WITHOUT_ANTREA_PROXY environment variable to true for the Antrea
-Agent. For example:
+Agent in the [Antrea deployment yaml](https://github.com/antrea-io/antrea/blob/main/build/yamls/antrea.yml).
```

Comment:
you should use a relative link here: `(../build/yamls/antrea.yml)`. IIRC, it will ensure the the link remains correct when generating the documentation for the antrea.io website.

Reply:
done, thanks for the reminder.